infrastructure/kctf/samples/apache-php/challenge/files/nsjail-php-cgi

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsjail --config /config/nsjail-php.cfg -- $@
```

Comment:
this command will be ran on every HTTP request. it could use "exec" to save some resources